[2024-07-30T11:31:22.207+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T11:31:22.309+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: enrich_prod.cleaning.check_esco_annexe_sensor manual__2024-07-30T11:31:17.990749+00:00 [queued]>
[2024-07-30T11:31:22.331+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: enrich_prod.cleaning.check_esco_annexe_sensor manual__2024-07-30T11:31:17.990749+00:00 [queued]>
[2024-07-30T11:31:22.332+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-30T11:31:22.369+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonSensor): cleaning.check_esco_annexe_sensor> on 2024-07-30 11:31:17.990749+00:00
[2024-07-30T11:31:22.384+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=3920) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-30T11:31:22.387+0000] {standard_task_runner.py:64} INFO - Started process 3934 to run task
[2024-07-30T11:31:22.387+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'enrich_prod', 'cleaning.check_esco_annexe_sensor', 'manual__2024-07-30T11:31:17.990749+00:00', '--job-id', '673', '--raw', '--subdir', 'DAGS_FOLDER/gps/custom_dags/enrich.py', '--cfg-path', '/tmp/tmpeeqh9idy']
[2024-07-30T11:31:22.393+0000] {standard_task_runner.py:91} INFO - Job 673: Subtask cleaning.check_esco_annexe_sensor
[2024-07-30T11:31:22.488+0000] {task_command.py:426} INFO - Running <TaskInstance: enrich_prod.cleaning.check_esco_annexe_sensor manual__2024-07-30T11:31:17.990749+00:00 [running]> on host 7d7c82e77e1e
[2024-07-30T11:31:22.833+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='richmond.kongo@orange.com,assane.diop@orange.com,cyrille1.danho@orange.com,mohamed2.sylla@orange.com,amadou.diomande@orange.com,Jonas.kouame@orange.com,armand.kablan@orange.com,anicet.azombade@orange.com,etienne.kouadio@orange.com,bertine.kouakou@orange.com,florentine.kouakou@orange.com,ange.phaceli@orange.com,soulegame.soro@orange.com,innocent.yengo@orange.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='enrich_prod' AIRFLOW_CTX_TASK_ID='cleaning.check_esco_annexe_sensor' AIRFLOW_CTX_EXECUTION_DATE='2024-07-30T11:31:17.990749+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-30T11:31:17.990749+00:00'
[2024-07-30T11:31:22.835+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T11:31:22.870+0000] {python.py:75} INFO - Poking callable: <function check_file at 0x798a999ab1a0>
[2024-07-30T11:31:52.751+0000] {local_task_job_runner.py:313} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-07-30T11:31:52.754+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-30T11:31:52.759+0000] {process_utils.py:132} INFO - Sending 15 to group 3934. PIDs of all processes in the group: [3934]
[2024-07-30T11:31:52.759+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 3934
[2024-07-30T11:31:52.760+0000] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-07-30T11:31:52.761+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T11:31:52.812+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=3934, status='terminated', exitcode=0, started='11:31:21') (3934) terminated with exit code 0
